engine/events.php

```diff
@@ -20,7 +20,6 @@
 	'cache:clear:after' => [
 		'system' => [
 			'\Elgg\Cache\EventHandlers::enable' => [],
-			'\Elgg\Cache\EventHandlers::rebuildPublicContainer' => [],
```

Comment:
isn't this needed anymore, or should it be done someplace else?

Reply:
probably no longer needed